app/src/main/java/com/example/bootcampproject/data/local/AvailableBooksDao.kt

```diff
@@ -1,9 +1,12 @@
 package com.example.bootcampproject.data.local
 
-import androidx.room.*
+import androidx.room.Dao
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy
+import androidx.room.Query
+import androidx.room.Update
 import com.example.bootcampproject.data.mock.AvailableBook
 
-
 @Dao
 interface AvailableBooksDao {
     @Query("Select * From AvailableBook")
```

Comment:
You can store table names in a constants file.

Reply:
done